tests/calculate.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+cd $1
+
+# - .vcf.gz files have no stochastic content except a header line with time information
+# - .tbi files are generically named, stochastic content which we are not checking
+# - .fin files are generically named, no stochastic content, may be md5sum-checked
+
+# Therefore:
+# - Check md5sums for all types of files, sort
+
+echo ".vcf files:"
+for v in *.vcf.gz;do gunzip -c $v | grep -v GATKCommandLine | md5sum;done | sort -V
+
+#echo ".tbi files:"
+#find . -name "*.tbi" | xargs md5sum | sort
```

Comment:
Rather than leaving this uncommented, which makes me think it's intending to be implemented later, can it either be replaced with a different check (is there any non-stochastic content?) or removed?

Reply:
I cannot think about a good way to check a binary file with stochastic content right now, but I can remove these two lines later.